core/dbsecurity/auth/Makefile

```diff
@@ -70,10 +67,11 @@ INCLUDES	= -I. -I./inc -I ../shared/inc \
 	        -I $(MY_SQROOT)/../sql/export \
 	        -I $(MY_SQROOT)/../sql/porting_layer \
 	        -I $(MY_SQROOT)/export/include \
+	        -I $(MY_SQROOT)/commonLogger \
 	        -I ../../sql/common
 
 
-LINK_OPTIONS	= -L$(LIBEXPDIR) -lldap -lssl -llber
+LINK_OPTIONS	= -L$(LIBEXPDIR) -lldap -lssl -llber -llog4cxx
 LINK_OPTIONS   += $(LNK_FLGS) 
 
 COMMON_LIBS =  -ltdm_sqlcli -larkcmp_dll  
```

Comment:
Makefile changes look fine. You might want to remove this COMMON_LIBS line since it is not actually being used.